### PR TITLE
[QA] Fix typo in fundrawtransaction test

### PR DESCRIPTION
**Pull request by NicolasDorier**
Ping @jnewbery introduced on https://github.com/bitcoin/bitcoin/commit/dab804c18a427901684ebe936b2069a97e04a268

### CODE REVIEW

**Comment by laanwj** [comment]
I don't get it, how did this typo pass the tests?

**Comment by NicolasDorier** [comment]
@laanwj becuse `rawtx` is an actual variable used above, which is planned to fail with this reason.

File: test/functional/fundrawtransaction.py [hunk 1]
```diff
@@ -472,7 +472,7 @@ def run_test(self):
         rawTx = self.nodes[1].createrawtransaction(inputs, outputs)
         # fund a transaction that requires a new key for the change output
         # creating the key must be impossible because the wallet is locked
-        assert_raises_jsonrpc(-4, "Insufficient funds", self.nodes[1].fundrawtransaction, rawtx)
```
Comment (dcousens) [inline test/functional/fundrawtransaction.py]: `rawtx` is also defined, see above?

**Comment by laanwj** [comment]
Oh wow. Two variables with the same name and different casing, this is confusing.

**Comment by jnewbery** [comment]
ouch. Mea culpa. Good spot @NicolasDorier.

Tested ACK 803e6a35037e7371f767585fec36ba17bdd36f4a

**Comment by laanwj** [comment]
I think the underlying problem should be solved too. We shouldn't use `rawTx` and `rawtx` in a single function, this is the code equivalent of a minefield.

**Comment by MarcoFalke** [comment]
utACK 803e6a3. Agree that the underlying problem should be solved as well. As a first step you can get rid of the camelCase and use a single name for the variable. In a second step, the scopes could be limited to the minimum necessary. (Maybe refactor the blocks into separate functions?)

**Comment by jonasschnelli** [comment]
utACK 803e6a35037e7371f767585fec36ba17bdd36f4a

**Comment by sipa** [comment]
utACK

**Review by dcousens — APPROVED** [review]

**Comment by MarcoFalke** [comment]
This has too much review to keep open any longer. Lets do the follow up cleanup in separate pulls.

**Comment by NicolasDorier** [comment]
Follow up https://github.com/bitcoin/bitcoin/pull/10083

**Pull request merged**